src/backend/tests/c-lisp/fibonacci.sexp

```diff
@@ -0,0 +1,14 @@
+(c-lisp
```

Comment:
also write a dynamic programming version of this as a test

Reply:
Done